presto-blackhole/src/test/java/io/prestosql/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -69,8 +69,14 @@ public void tearDown()
     public void testCreateSchema()
     {
         assertEquals(queryRunner.execute("SHOW SCHEMAS FROM blackhole").getRowCount(), 2);
+        assertThatQueryReturnsValue("CREATE TABLE nation as SELECT * FROM tpch.tiny.nation", 25L);
+
         queryRunner.execute("CREATE SCHEMA blackhole.test");
         assertEquals(queryRunner.execute("SHOW SCHEMAS FROM blackhole").getRowCount(), 3);
+        assertThatQueryReturnsValue("CREATE TABLE test.nation as SELECT * FROM tpch.tiny.nation", 25L);
```

Comment:
Since default.nation and test.nation are identical, we cannot test whether we have 2 tables or just one.
Create tables with same name, but with different "type" (column list).